gini/gini-vision-lib-ios
# Update camera screen UI

Camera screen UI has been redesign, changing not only icon or colors, but also how the guides are shown (Not an overlay image but guides drawn programmatically).
Also some error messages has been added, being more specific now.

### How to test
With an iPad try all possible orientations (also with split mode) and see that the preview guides are resized properly. 
Also try to import an invalid file and see the error shown.

### Merging
Automatic.

### Ticket 
Issue #65
